crates/host/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "wasmcloud-host"
-version = "0.23.1"
+version = "0.24.0"
```

Comment:
Nit: Couldn't we technically get away with a patch bump? Changes in dep crates are always interesting. Nothing changes in the API of the crate, so I think it could be fine.

Either way, I don't care what we do, was just calling it out

Reply:
We certainly can if that's preferred, I don't have a strong preference in this case, a minor bump felt appropriate given the changes, but we can go with patch.

Reply:
I've switched it to `0.23.2`